lms/static/scripts/frontend_apps/components/OAuth2RedirectErrorApp.js

```diff
@@ -24,18 +25,17 @@ export default function OAuth2RedirectErrorApp({ location = window.location }) {
   const { OAuth2RedirectError = /** @type {OAuthErrorConfig} */ ({}) } =
     useContext(Config);
 
-  const {
-    authUrl = null,
-    errorCode = null,
-    errorDetails = '',
-    canvasScopes = /** @type {string[]} */ ([]),
-  } = OAuth2RedirectError ?? {};
+  const { authUrl = null, canvasScopes = /** @type {string[]} */ ([]) } =
+    OAuth2RedirectError ?? {};
```

Comment:
This was an existing issue, but I notice the `{}` fallback is present both here and in the `useContext` call above. We should only need it in one place really, probably here.